### ISSUE TITLE
Modularize Out MkDocs Material Theming/Customization

### ISSUE THREAD

**Issue by Sewer56** [Nexus-Mods/NexusMods.Archives.Nx#1]
## Currently 
This project is using MkDocs-Material, which has been customized to feature the theme of 'NexusMods Next'. However, if left in this repository, this customized code will eventually end up being copied copied across multiple repositories, leading to inefficiency and increased maintenance overhead.

## We want
We would like to move all of the customizations related to the 'NexusMods Next' theme into a single repository. This would help streamline the development process and make it easier to manage and update the customizations in one place.

## Supporting Information
Here is a list of repositories where the customized MkDocs-Material theme is currently being used:
1. This Repository
2. NexusMods.Paths (possibly in future)
3. NexusMods.BenchmarkFramework (soon)

## Design
To implement this improvement:

1. Create a new repository to host the 'NexusMods Next' theme customizations.
2. Migrate all existing customizations from the respective repositories to the new repository.
3. Update the MkDocs-Material configuration in thisrepository to reference the new centralized theme repository.
    - In other words, use a submodule.
5. Test and verify that the customizations are working correctly.
